actions/testmo-run-create/action.yml

```diff
@@ -1,27 +1,25 @@
+---
 name: create testmo run
-description: 'create neuralmagic testmo run and return its ID'
+description: create neuralmagic testmo run and return its ID
+
 inputs:
   testmo_url:
-    description: 'testmo URL'
+    description: testmo URL
     required: true
   testmo_token:
-    description: 'testmo token'
+    description: testmo token
     required: true
   source:
-    description: "source for testmo, e.g. 'build-test'"
+    description: source for testmo, e.g. 'build-test'
     required: true
-  project_id:
```

Comment:
project_id is needed. Please rebase from latest main.

Reply:
okay, looks like a bad merge on my part ... :)